module/data/item/race.mjs

```diff
@@ -0,0 +1,73 @@
+import SystemDataModel from "../abstract.mjs";
+import { AdvancementField, IdentifierField } from "../fields.mjs";
+import ItemDescriptionTemplate from "./templates/item-description.mjs";
+
+/**
+ * Data definition for Race items.
+ * @mixes ItemDescriptionTemplate
+ *
+ * @property {string} identifier       Identifier slug for this race.
+ * @property {object[]} advancement    Advancement objects for this race.
+ * @property {number} darkvision       Darkvision distance.
+ * @property {object} movement
+ * @property {number} movement.burrow
+ * @property {number} movement.climb
+ * @property {number} movement.fly
+ * @property {number} movement.swim
+ * @property {number} movement.walk
+ * @property {string} movement.units   Units used to measure movement.
+ * @property {object} type
+ * @property {string} type.value       Creature type (e.g. "humanoid", "construct").
+ * @property {string} type.subtype     Additional type data (e.g. "elf" for Eladrin)
+ */
+export default class RaceData extends SystemDataModel.mixin(ItemDescriptionTemplate) {
+  /** @inheritdoc */
+  static defineSchema() {
+    return this.mergeSchema(super.defineSchema(), {
+      identifier: new IdentifierField({label: "DND5E.Identifier"}),
+      advancement: new foundry.data.fields.ArrayField(new AdvancementField(), {label: "DND5E.AdvancementTitle"}),
+      darkvision: new foundry.data.fields.NumberField({min: 0, integer: true, label: "DND5E.SenseDarkvision"}),
+      movement: new foundry.data.fields.SchemaField({
```

Comment:
It would be nice to have this share the same movement schema that an actor has, similar to my comment about senses above. If we have a `MovementData` model we can embed it here like:
```js
movement: new fields.SchemaField(MomvementData.defineSchema())
```
Or as:
```js
movement: new fields.EmbeddedDataField(MomvementData)
```
Which is a bit more powerful, but also slightly heavier for performance.

Reply:
Yeah another +1 here that we should reuse senses and movement schema and allow races to provide more than just darkvision as a sense.